index.html

```diff
@@ -124,82 +124,82 @@ <h2 class="text-center">My Journey</h2>
           <hr class="mg-btm" />
           <p>
             I started my life wanting to be a professional football player. When
-            i got scouted by VFL Wolfsburg i believed that my dream would come
+            I got scouted by VFL Wolfsburg I believed that my dream would come
             true. But after playing for VFL Wolfsburg for one year and for RB
-            Leipzig for 2 years, i had to accept the fact that i will not become
+            Leipzig for 2 years, I had to accept the fact that I will not become
             a major football player. Unsure on what is the right professional
-            fit for me i dived into several different paths of educations and
-            work. Until i finally came across programming and felt that this was
-            my passion. In the following months i consumed a lot of tutorials
+            fit for me I dived into several different paths of educations and
+            work. Until I finally came across programming and felt that this was
+            my passion. In the following months I consumed a lot of tutorials
             and courses online. Currently I am taking the bootcamp by neuefische
             and I am excited about what follows.
           </p>
         </div>
       </section>
 
-      <section class="bg-cl-yellow">
+      <section class="cv bg-cl-yellow">
         <div class="container">
           <h2 class="text-center">Experiences</h2>
           <hr class="mg-btm" />
           <h3>Educations</h3>
-          <div class="exp-wrap">
+          <div class="cv__exp-wrap">
```

Comment:
Could you not give this class to the `ul`? This `div` seems a bit unnecessary. Also, I would encourage you to not shorten class names.